public/pages/CreateTransform/containers/CreateTransformForm/CreateTransformForm.tsx

```diff
@@ -213,6 +219,13 @@ export default class CreateTransformForm extends Component<CreateTransformFormPr
     let currentStep = this.state.currentStep;
     let warned = this.state.beenWarned;
     let error = false;
+
```

Comment:
These three lines are also repeated across function, can we define a common function in MDSEnabledComponent ?

Reply:
These three lines are used only in two components whereas MDSEnableComponent is being inherited by all the components. I think this is not a common functionality across all components which need not to be abstracted out. Feel free to unresolve and continue if you think otherwise